File: Client/Assets/iTweenExamples/Platforms/Scripts/InfiniteBackground.js
public var scrollSpeed : float;
private var cam : Transform;
private var cameraInit : Vector3;

function Start (){
	cam=Camera.main.transform;
	cameraInit=new Vector3(cam.position.x,cam.position.y,cam.position.z);
}

function FixedUpdate (){
	var difference : Vector3=cameraInit - cam.position;
	GetComponent.<Renderer>().material.mainTextureOffset = difference*scrollSpeed;
}

File: Client/Assets/Package/Pixelplacement/iTweenExamples/Platforms/Scripts/InfiniteBackground.cs
using UnityEngine;
using System.Collections;

public class InfiniteBackground : MonoBehaviour{
	public float scrollSpeed;
	Transform cam;
	Vector3 cameraInit;

	void Start (){
		cam=Camera.main.transform;
		cameraInit=new Vector3(cam.position.x,cam.position.y,cam.position.z);
	}

	void FixedUpdate (){
		Vector3 difference=cameraInit - cam.position;
		GetComponent<Renderer>().material.mainTextureOffset = difference*scrollSpeed;
	}
}